components/modal/match/MatchManualModal.tsx

```diff
@@ -3,34 +3,120 @@ import { useSetRecoilState } from 'recoil';
 import { MatchMode } from 'types/mainType';
 import { Manual } from 'types/modalTypes';
 import { modalState } from 'utils/recoil/modal';
-import ModeToggle from 'components/mode/modeItems/ModeToggle';
+import ModeRadiobox from 'components/mode/modeItems/ModeRadiobox';
 import styles from 'styles/modal/match/MatchManualModal.module.scss';
 
+type contentType = {
+  title: string;
+  description: string[];
+};
+
+type contentsType = Record<MatchMode, contentType[]>;
+
+// TODO : 전체 타입에 대한 설명 추가해야 합니다.
+const modalContents: contentsType = {
+  BOTH: [
+    {
+      title: '😳 아직 정해지지 않음',
```

Comment:
How about changing the title type to receive it as a React.Reactnode instead of a string so that we can handle that part as a jsx.element? Then, you can use the emoji as SVG file.

Reply:
오... 이모지 쓰는게 신경쓰였었는데 좋은 생각인 것 같아요! 바꿔보겠습니다!